tests/other_test.go

```diff
@@ -22,3 +25,63 @@ func TestRunBasicExample(t *testing.T) {
 	assert.Nil(t, err, "This should not have errored")
 	assert.NotNil(t, output, "Expected some output")
 }
+
+func TestRunCompleteOtherVersionExample(t *testing.T) {
+	t.Parallel()
+	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
+		Testing:       t,
+		TerraformDir:  completeExampleTerraformDir,
+		Prefix:        "etcd-fscloud",
+		Region:        "us-south",
+		ResourceGroup: resourceGroup,
+		TerraformVars: map[string]interface{}{
+			// "access_tags":                permanentResources["accessTags"],
+			// "existing_kms_instance_guid": permanentResources["hpcs_south"],
+			// "kms_key_crn":                permanentResources["hpcs_south_root_key_crn"],
+			"etcd_version": "3.4", // This test will run using version 3.4
+		},
+	})
+	options.SkipTestTearDown = true
+	output, err := options.RunTestConsistency()
+	assert.Nil(t, err, "This should not have errored")
+	assert.NotNil(t, output, "Expected some output")
+
+	// check if outputs exist
+	outputs := terraform.OutputAll(options.Testing, options.TerraformOptions)
+	expectedOutputs := []string{"port", "hostname"}
+	_, outputErr := testhelper.ValidateTerraformOutputs(outputs, expectedOutputs...)
+	assert.NoErrorf(t, outputErr, "Some outputs not found or nil")
+	options.TestTearDown()
+}
+
+func TestRunCompleteExampleOtherVersion(t *testing.T) {
+	t.Parallel()
+
+	// Generate a 15 char long random string  for the admin_pass
+	randomBytes := make([]byte, 13)
+	rand.Read(randomBytes)
+	randomPass := "A1" + base64.URLEncoding.EncodeToString(randomBytes)[:13]
+
+	options := testhelper.TestOptionsDefaultWithVars(&testhelper.TestOptions{
+		Testing:            t,
+		TerraformDir:       completeExampleTerraformDir,
+		Prefix:             "etcd-complete",
+		ResourceGroup:      resourceGroup,
+		BestRegionYAMLPath: regionSelectionPath,
+		TerraformVars: map[string]interface{}{
+			"etcd_version": "3.4",
+			"users": []map[string]interface{}{
+				{
+					"name":     "testuser",
+					"password": randomPass, // pragma: allowlist secret
+					"type":     "database",
+				},
+			},
+			"admin_pass": randomPass,
+		},
+	})
+
+	output, err := options.RunTestConsistency()
+	assert.Nil(t, err, "This should not have errored")
+	assert.NotNil(t, output, "Expected some output")
+}
```

Comment:
We have two tests in pr_test.go, so they can test 3.4 and 3.5. So we only need 1 test here to test 3.3 and that can be the basic example. No need to add any extra tests here

Reply:
removed the extra test added and locked the version to 3.3 for this test.